runtime/gc_base/RootScanner.cpp

```diff
@@ -395,44 +395,44 @@ MM_RootScanner::scanPermanentClasses(MM_EnvironmentBase *env)
 {
 	reportScanningStarted(RootScannerEntity_PermanentClasses);
 
+	/* Do systemClassLoader */
+	scanClassloader(env, static_cast<J9JavaVM*>(_omrVM->_language_vm)->systemClassLoader);
+
+	/* Do applicationClassLoader */
+	scanClassloader(env, static_cast<J9JavaVM*>(_omrVM->_language_vm)->applicationClassLoader);
```

Comment:
Does the extension (platform) classloader also need to be scanned?

Reply:
The extension/platform classloader can't be unloaded. It's the parent of the application classloader.